### ISSUE TITLE
tesT_p

### ISSUE THREAD

**Issue by bwplotka** [GoogleCloudPlatform/prometheus-engine#704]
0.9.0 rule with `project_id=gmp-eval-central` selector fails to be applied:
 
```
Error from server (isolating rules failed: set isolation selector project_id="test_project" on up{cluster="gmp-test-recording-rules",project_id="gmp-eval-central"}: conflicting label matcher project_id="gmp-eval-central" found): error when creating "rules.yaml": admission webhook "[validate.rules.gmp-operator.gmp-system.monitoring.googleapis.com](http://validate.rules.gmp-operator.gmp-system.monitoring.googleapis.com/)" denied the request: isolating rules failed: set isolation selector project_id="test_project" on up{cluster="gmp-test-recording-rules",project_id="gmp-eval-central"}: conflicting label matcher project_id="gmp-eval-central" found
```

**Comment by TheSpiritXIII** [comment]
My 2-cents since we saw a customer report this issue recently: besides maybe rewording this error message, maybe we also shouldn't make it a hard error if they're using the correct values?
